tests/monotouch-test/monotouch-test.csproj

```diff
@@ -761,7 +762,6 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="Security\openssl_crt.der">
-        <LogicalName>monotouchtest.Security.openssl_crt.der</LogicalName>
```

Comment:
I think the IDE is technically right (it's removing the value because it's identical to the default value), but xharness currently needs the value when creating test variations (watchOS/tvOS projects).